js/entity_browser.common.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * @file entity_browser.common.js
+ *
+ * Common helper functions used by various parts of entity browser.
+ */
+(function ($, Drupal, drupalSettings) {
+
+  "use strict";
+
+  Drupal.entityBrowser = {};
+
+  /**
+   * Reacts on "entities selected" event.
+   *
+   * @param event
+   *   Event object.
+   * @param uuid
+   *   Entity browser UUID.
+   * @param entities
+   *   Array of selected entities.
+   */
+  Drupal.entityBrowser.selectionCompleted = function(event, uuid, entities) {
```

Comment:
This two functions lived in namespaces owned by specific parts of entity browser. It turns out they can be useful outside of their original scope so let's move them here.